src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java

```diff
@@ -247,14 +247,15 @@ public static HologramData readHologram(String name, ConfigurationSection config
                 default -> TextDisplay.TextAlignment.CENTER;
             };
 
-            TextColor background = null;
+            Color background = null;
             if (backgroundName != null) {
                 if (backgroundName.equalsIgnoreCase("transparent")) {
                     background = Hologram.TRANSPARENT;
                 } else if (backgroundName.startsWith("#")) {
-                    background = TextColor.fromHexString(backgroundName);
+                    background = Color.fromARGB((int)Long.parseLong(backgroundName.substring(1), 16));
```

Comment:
Why do you parse it as long and then cast to integer?

Reply:
> Why do you parse it as long and then cast to integer?

It was throwing some exception for number being too large or it was something to do with negativity of the number I can't tell right now but you can try with `Integer.parseInt` and see the error yourself.